internal/engine/upper.go

```diff
@@ -184,6 +185,8 @@ func upperReducerFn(ctx context.Context, state *store.EngineState, action store.
 		handleMetricsDashboardAction(state, action)
 	case server.OverrideTriggerModeAction:
 		handleOverrideTriggerModeAction(ctx, state, action)
+	case apiserver.SyncAction:
```

Comment:
is this still used?

Reply:
Ah, nope - was left over from POC + ensuring import cycle insanity was resolved. Removed!